_posts/0000-01-03-ericalloyd.md

```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Hey there! I'm Constructocat v2.0!
 
 ![constructocat](https://octodex.github.com/images/constructocat2.jpg)
```

Comment:
If you don't like your image you can always find more here: https://octodex.github.com